staging/src/k8s.io/apiserver/pkg/storage/storagebackend/config.go

```diff
@@ -49,6 +49,7 @@ type Config struct {
 
 func NewDefaultConfig(prefix string, copier runtime.ObjectCopier, codec runtime.Codec) *Config {
 	return &Config{
+		Type:   StorageTypeETCD3,
```

Comment:
I'm fine with that, but then please remove the defaulting from that other place (and tread undefined there as error).